makerspace/templates/tools/admin_tool.html

```diff
@@ -0,0 +1,40 @@
+{% extends "web/base.html" %}
+{% load static %}
+{% load i18n %}
+
+{% block body %}
+    <h1 class="ui text-centered make_col_yellow header">
+        {% trans "Tools" %}
+    </h1>
+    <div class="ui text container">
+        <a class="ui yellow button" href="{% url 'makerspace-tools-create' %}">
+            <i class="ui add icon green"></i>
+            {% trans "Add new tool" %}
+        </a>
+        {% if tools_list %}
+            <div class="ui divided list">
+                {% for tool in tools_list %}
+                    <div class="item">
+                        <i class="ui icon"><img class="inline square_36" src="{{ tool.image.url }}"></i>
+                        <div class="content">
+                            <div class="header">
+                                <a class="inline-block make_col_blue header" href="{% url 'makerspace-tools-edit' tool.id %}">
+                                    {{ tool.title }}
+                                </a>
+                                {% if perms.makerspace.delete_tool %}
+                                    <a class="delete confirm inline-block make_col_blue explanation-popup"
```

Comment:
This popup is not shown.

Reply:
It does show up for me
![image](https://user-images.githubusercontent.com/43437044/82940439-1f29d480-9f95-11ea-961b-023077214719.png)

Reply:
I thought there was another popup, as only `delete confirm` is required for the deletion confirmation modal.

Reply:
Has this been resolved, @hermabe?

Reply:
`inline-block make_col_blue explanation-popup` is not needed for the delete modal. If there is another popup that should show it must be fixed. If not, remove the uneccessary classes.

Reply:
They've been removed in c8b990c3ae481c4c5bdc61fb59e106efbeb676c8 🙂 Resolving conversation.